<?xml version="1.0"?>

<launch>
  	<arg name="debug" default="false" />
  	<arg name="node_name" default="schunk_ft_sensor" />
  	<arg name="can_device"/>
  	<arg name="can_node_id"/>
  	<arg name="publish_rate" default="50.0"/>
  	<arg name="silence_limit" default="0.100"/>
  	
	<node name="$(arg node_name)" pkg="schunk_ft_sensor" type="schunk_ft_sensor_node" output="screen">
		<param name="debug" value="$(arg debug)"/>
		<param name="can_device" value="$(arg can_device)"/>
		<param name="can_node_id" value="$(arg can_node_id)"/>
		<param name="publish_rate" value="$(arg publish_rate)"/>
		<param name="silence_limit" value="$(arg silence_limit)"/>
	</node>

</launch>